Makefile

```diff
@@ -1,16 +1,18 @@
 .PHONY: ui vendor build ui-agent ui-manager container-tornjak-be-spire container-tornjak-be-spire-push container-manager container-manager-push release-tornjak-be-spire-multiversions push container-frontend container-frontend-push container-tornjak-be container-tornjak-be-push
 
+VERSION=$(shell cat version.txt)
+
 CONTAINER_TAG ?= tsidentity/tornjak:latest
 CONTAINER_BACKEND_TAG ?= tsidentity/tornjak-be:latest
 CONTAINER_BACKEND_WITH_SPIRE_TAG ?= tsidentity/tornjak-be-spire-server:latest
 CONTAINER_FRONTEND_TAG ?= tsidentity/tornjak-fe:latest
```

Comment:
We should also add these new tags to images created for development (tsidentity/xxxx)

Reply:
done